packages/components/src/date-time/date/test/use-lilius.ts

```diff
@@ -0,0 +1,446 @@
+/**
+ * This source is a local copy of the use-lilius library.
+ *
+ * use-lilius@2.0.5
+ * https://github.com/its-danny/use-lilius
```

Comment:
This URL seems to redirect to https://github.com/Avarios/use-lilius. Which URL is appropriate? @its-danny

Reply:
Yeah, I transferred ownership to them, so that one is fine.

Reply:
The reson why I used that URL is that the library author in the [discussion](https://github.com/WordPress/gutenberg/discussions/64968) pointed to `its-danny/use-lilius` — and the author listed in the MIT license (Danny Tatom) also coincides with that.

Reply:
> Yeah, I transferred ownership to them, so that one is fine.

@its-danny , just to make sure — are you suggesting to use `https://github.com/Avarios/use-lilius` instead?

Reply:
Updated to `https://github.com/Avarios/use-lilius` - we can always tweak this in a future PR if needed.